apps/hubble-stats/containers/NetworkTableContainer/NetworkTableContainer.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
+
+import { FC, useState } from 'react';
+import { TableAndChartTabs, Typography } from '@webb-tools/webb-ui-components';
+
+import { NetworkTable } from '../../components';
+import { TokenCompositionType } from '../../components/NetworkTable/types';
+
+const NetworkTableContainer: FC = () => {
+  const [chains, setChains] = useState<number[]>([]);
+  const [tvlData, setTvlData] = useState<TokenCompositionType[]>([]);
+  const [volumeData, setVolumeData] = useState<TokenCompositionType[]>([]);
+  const [depositsData, setDepositsData] = useState<TokenCompositionType[]>([]);
+  const [feesData, setFeesData] = useState<TokenCompositionType[]>([]);
```

Comment:
It would be a good idea to transform this component into a server component. You could do this by eliminating all instances of useState. To emulate data fetching, you can create a data fetching function outside of the component, which returns dummy data.

Next, transform this component into an asynchronous one and call the newly created data fetching function to obtain data. Once you have the data, you can pass it into the child components for rendering. This way, you'll maintain a smoother and more efficient component structure.